v-next/hardhat-mocha/src/type-extensions.ts

```diff
@@ -3,19 +3,22 @@ import "hardhat/types/config";
 import type { MochaOptions } from "mocha";
 
 declare module "hardhat/types/config" {
-  export interface HardhatUserConfig {
-    mocha?: MochaOptions;
-  }
-
   export interface TestPathsUserConfig {
     mocha?: string;
   }
 
-  export interface HardhatConfig {
-    mocha: MochaOptions;
-  }
-
   export interface TestPathsConfig {
     mocha: string;
   }
 }
+
+import "hardhat/types/test";
```

Comment:
I was unsure whether these type should be under `hardhat/types/config` or `hardhat/types/test`